ui/admin/app/routes/scopes/scope/add-storage-policy/index.js

```diff
@@ -22,13 +22,20 @@ export default class ScopesScopeAddStoragePolicyIndexRoute extends Route {
    */
   async afterModel() {
```

Comment:
I just noticed this but how come we do this in the `afterModel` and not just the `model`?

Reply:
so we want this to be in the `afterModel` method as this is part of the scope model, any `abandon confirmation`, or `cancellation` is based on the scope model and not policies